esx_service/vmdk_ops.py

```diff
@@ -590,6 +600,7 @@ def disk_attach(vmdk_path, vm):
     spec = vim.vm.ConfigSpec()
     spec.deviceChange = disk_changes
 
+    # Handle the most generic exception for a VM reconfig.
```

Comment:
I doubt if this comment is giving reader any addition information. I suggest dropping it from here and other places.